meta/validation.json

```diff
@@ -97,5 +70,32 @@
                 }
             ]
         }
+    },
+    "$defs": {
```

Comment:
I don't know... I like `$defs` earlier so that I know a thing when it's referenced.

Reply:
Having worked with a large set of schemas, often with large definitions sections, it was maddening trying to find the rest of the top-level schema for me.

I eventually got fed up and hand-reformatted around 60 schema files, IIRC.  It was that annoying.  `$defs` is a single keyword you can search for, so it's really easy to jump to.

I'm not going to fight to the death over this, just making the point that it wasn't a coin flip.

Reply:
I see it analogous to coding in C, where you have to declare your variables before you use them.  I can see your point of making it more like an appendix, though, too.

Meh.

Reply:
I kind of see it like an embedded library.  I typically want to see the main structure first, and then drill down into what libraries it uses.  Sometimes I don't care about the details, so
I might have reason to trust that `"#/$defs/postalAddress"` does what I want, but I definitely want to see where it is used.

I also sometimes split schemas from `$defs` out into separate files, and it's nice (to me anyway) that the top section doesn't change beyond updating the `$ref` when that happens.

Reply:
I tend to agree with having `"$defs"` after the main structure.

Reply:
My personal preference would be for at the start, but it's a very weekly held preference. At the end is also fine.